packages/plugin-conversation/src/process-image.browser.js

```diff
@@ -52,7 +52,7 @@ export function computeDimensions({width, height}, maxWidth, maxHeight) {
  * @param {Number} thumbnailMaxHeight
  * @returns {Promise<Array>} Buffer, Dimensions, thumbnailDimensions
  */
-export default function processImage(file, thumbnailMaxWidth, thumbnailMaxHeight) {
+export default function processImage({file, thumbnailMaxWidth, thumbnailMaxHeight, enableThumbnails}) {
```

Comment:
Let's add documentation for this new param and the update to using a single object argument